dragonfly_schema/cli/__init__.py

```diff
@@ -0,0 +1,73 @@
+"""
+Command Line Interface (CLI) entry point for dragonfly-schema.
+
+Note:
+    Do not import this module in your code directly. For running the commands,
+    execute them from the command line or as a subprocess (e.g.
+    ``subprocess.call(['dragonfly-schema', 'viz'])``)
+
+Honeybee is using click (https://click.palletsprojects.com/en/7.x/) for creating the CLI.
+"""
+
+try:
+    import click
+except ImportError:
+    raise ImportError('click module is not installed.')
+
+import sys
+import os
+import logging
+import json
+
+from dragonfly_schema.model import Model
+
+
+_logger = logging.getLogger(__name__)
+
+
+@click.group(help='Commands for validating JSON files.')
```

Comment:
I assume you will probably have commands more than just the validating commands. For instance you might add add a command to translate the schema to an energy model or a radiance model, etc.